internal/apiserver/root.go

```diff
@@ -46,14 +45,10 @@ const PGOSecretName = "pgo.tls"
 const VERSION_MISMATCH_ERROR = "pgo client and server version mismatch"
 
 var (
-	// PGOClientset is a client for PostgreSQL Operator resources
-	PGOClientset pgov1.Interface
-	// Clientset is a client for native Kubernetes resources
-	Clientset kubernetes.Interface
+	// Clientset is a client for Kubernetes resources
+	Clientset kubeapi.Interface
 	// RESTConfig holds the REST configuration for a Kube client
 	RESTConfig *rest.Config
-	// RESTClient is a REST client for the Kubernetes API
-	RESTClient *rest.RESTClient
```

Comment:
Here's where the clients changed in apiserver. They are initialized in the last hunk of this same file.